Formula/apr.rb

```diff
@@ -49,14 +50,37 @@ def install
     system "./configure", "--prefix=#{libexec}"
     system "make", "install"
     bin.install_symlink Dir["#{libexec}/bin/*"]
+    lib.install_symlink Dir["#{libexec}/lib/*.a"]
+    on_macos do
+      lib.install_symlink Dir["#{libexec}/lib/*.dylib"]
```

Comment:
This should work (I think we already used it somewhere else): `lib.install_symlink Dir["#{libexec}/lib/#{shared_library("*")}"]`

Reply:
It seems it only supports `*.so` files, and not `*.so.*` files: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/os/linux/formula.rb#L7-L9

`ipopt` uses `shared_library` and, I'm not sure but it seems to be missing some lib files, notably `libmpiseq.so.*` and `libpord.so.*`

`ipopt` bottle: https://bintray.com/linuxbrew/bottles/ipopt/3.13.3_2#files
`ipopt` formula install: https://github.com/Homebrew/homebrew-core/blob/master/Formula/ipopt.rb#L54-L58

Reply:
Cool. Let's merge that and use it :)